src/test/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/validation/ParticipationValidatorTest.java

```diff
@@ -6,19 +6,24 @@
 import ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.entity.Game;
 import ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.entity.GameState;
 import ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.entity.Participation;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
 import org.springframework.web.client.HttpClientErrorException;
 
 class ParticipationValidatorTest {
+  private static final int MAX_NUMBER_OF_PLAYERS = 5;
+
```

Comment:
Detail: to me the test class would have more impression of reliability, if we instantiated the game in the BeforeEach section every time new. So no old values are carried over to from one test method to the next.